apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -88,11 +88,21 @@ ClassroomList.propTypes = {
   selectedId: React.PropTypes.string,
 };
 
+const LoadError = () =>
+  <div>
+    <a href="//localhost-studio.code.org:3000/users/auth/google_oauth2?scope=classroom.courses.readonly,classroom.rosters.readonly">
+      {locale.authorizeGoogleClassrooms()}
+    </a>
+  </div>;
+
 export default class RosterDialog extends React.Component {
   static propTypes = {
     handleClose: React.PropTypes.func,
     isOpen: React.PropTypes.bool,
-    classrooms: React.PropTypes.arrayOf(classroomShape),
+    classrooms: React.PropTypes.oneOf(
+      React.PropTypes.arrayOf(classroomShape),
+      React.PropTypes.bool,
```

Comment:
This feels like a weird type definition. Could we instead have classrooms be null or undefined?